src/files/test_test_file_methods.py

```diff
@@ -0,0 +1,50 @@
+"""Methods of File Objects
+
+@see: https://docs.python.org/3/tutorial/inputoutput.html#methods-of-file-objects
+
+Reading from a file does not always have to be sequential. There are methods to look for
+specific locations in the file, much like flipping to a page in a book.
+"""
+
+
+def test_file_methods():
+    """Methods of File Objects"""
+
+    multi_line_file = open('src/files/test_multi_line_file.txt', 'r')
+    binary_file = open('src/files/binary_file', 'r')
```

Comment:
why we are not opening it in binary mode(`rb`)?

Reply:
currently the file "binary file" was supposed to have entries like a binary number i.e. 0s and 1s only. our intention here was to read that file as a text file with binary numbers as it is. for this purpose I was opening the binary file as read only to read the text as string.